Tribler/Core/Modules/restapi/events_endpoint.py

```diff
@@ -40,21 +43,35 @@ def write_data(self, message):
         else:
             self.events_request.write(message)
 
+    def start_new_query(self):
+        self.infohashes_sent = set()
```

Comment:
Then set them to None and initialize when you have to. That's how it should be done in the python way right?